docs/pages/docs/reference/command-line-reference.mdx

```diff
@@ -75,6 +75,18 @@ Run NPM scripts across all packages in specified scope. Tasks must be specified
 to the tasks to be executed. Note that these additional arguments will _not_ be passed to
 any additional tasks that are run due to dependencies from the [pipeline](/docs/reference/configuration#pipeline) configuration.
 
+Uses glob patterns via [`doublestar`](https://github.com/bmatcuk/doublestar) under the hood. Specifying multiple glob patterns adds to the result.
```

Comment:
I wouldn't call out `doublestar` specifically here, since we are not necessarily always going to be using that library. The feature list below is a better solution, since it describes what we will support in the event that we do change our globbing library